gradle.properties

```diff
@@ -1,3 +1,4 @@
+org.gradle.caching=true
```

Comment:
Does this option in our repository affects our user?
It seems that it can affect different Kotlin builds, so I don't think, that it is a good idea to make this change

Reply:
You're right, this affect the build of the Kotlin plugin itself, not users of the Kotlin plugin. I'll drop it from the commit, although it might make sense for you to enable this anyway, to speed up the build of the Kotlin plugin itself.